api/modules/feedback/views.py

```diff
@@ -30,3 +31,44 @@ def add_feedback(request):
 
     success_message = "Sucessfully added new feedback."
     return Response(success_message, status=status.HTTP_201_CREATED)
+
+
+@api_view(['GET'])
+def get_feedback_id(request, feedback_id):
+    """
+    Returns the feedback pertaining to a certain feedback id
+    :param request:
+    :return: 400 if incorrect parameters are sent or database request failed
+    :return: 201 successful
+    """
+    try:
+        user_feedback = Feedback.objects.get(pk=feedback_id)
+
+    except Feedback.DoesNotExist:
+        error_message = "Feedback doesnt exist"
```

Comment:
Same, remove this

Reply:
But this is the second API that was asked in the issue

Reply:
Remove the error message

Reply:
cool